_commands/basics/bg.md

```diff
@@ -23,15 +23,24 @@ $
 
 ~~~ bash
 $ jobs
+$ sleep 100
+^Z
+$ gedit file.txt
+^Z
+$ jobs
 [1]+ Stopped    sleep 100
 [2]+ Stopped    gedit file.txt
 $ bg 2
 $ jobs
 [1]+ Stopped    sleep 100
 [2]+ Running    gedit file.txt
+$ bg 'sleep 100'
+$ jobs
+[1]+ Running    sleep 100
+[2]+ Running    gedit file.txt
 $
```

Comment:
Could also be useful to show how to background multiple jobs at once.

Reply:
@cameron-gagnon so added